### PR TITLE
Kkwiek/add info of images

### PR DESCRIPTION
**Pull request by mutirri**

### CODE REVIEW

File: web/source/miniconda.rst [hunk 1]
```diff
@@ -80,6 +80,14 @@ update``. For instance
 
 will update conda.
 
+.. _images:
+
+Try images with miniconda for cloud services
+--------------------------------------------
+
+More information about this topic You can find `here <http://docs.continuum.io/anaconda/images.html>`_
```
Comment (asmeurer) [inline web/source/miniconda.rst]: "You" should not be capitalized.

File: web/source/index.rst [hunk 2]
```diff
@@ -126,6 +126,12 @@ subscribe to the `conda mailing list
 <https://groups.google.com/a/continuum.io/forum/#!forum/conda>`_.  The source
 code and issue tracker for conda are on `GitHub <https://github.com/conda/conda>`_.
 
+Try images with miniconda for cloud services
+--------------------------------------------
+
+If You are using cloud services and want to very easily check Miniconda through one of
+already prepared by Us images, then please read `this information <http://docs.continuum.io/anaconda/images.html>`_
```
Comment (asmeurer) [inline web/source/index.rst]: This has some grammar issues, but anyway, I don't think we need to have it both on the main page and the miniconda page.

File: web/source/miniconda.rst (hunk 1 above)
Comment (asmeurer) [inline web/source/miniconda.rst]: "You can find more information about this topic here"

Comment (mutirri) [inline web/source/miniconda.rst]: Corrected. Thanks.

File: web/source/index.rst (hunk 2 above)
Comment (mutirri) [inline web/source/index.rst]: The sentence was rewritten.
And decision is up to you, which one should be deleted?

**Comment by asmeurer** [comment]
I fixed a grammar issue. This should be deployed as soon as Travis CI runs.

**Pull request merged**